.eslintrc

```diff
@@ -1,24 +1,24 @@
 {
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "module",
-    "ecmaFeatures": {
-      "modules": true
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "modules": true
+        }
+    },
+    "rules": {
+        "no-multiple-empty-lines": ["error", { "max": 1 }],
+        "space-before-function-paren": 0,
+        "react/jsx-fragments": 0,
+        "import/export": 0,
+        "max-len": [
+            "error",
+            { "code": 120, "ignoreTemplateLiterals": true, "ignoreStrings": true, "ignoreComments": true }
```

Comment:
I don't know why the formatting here changed so much, but the only change in this file is this `max-len` rule.

https://eslint.org/docs/rules/max-len